deeprank_gnn/models/graph.py

```diff
@@ -1,16 +1,21 @@
 from enum import Enum
-from typing import Callable, Union, List, Dict, Optional
-from uuid import uuid4
+from typing import Callable, Union, List
 import logging
 
 import numpy
 import h5py
 
-from deeprank_gnn.models.structure import Atom, Residue
-from deeprank_gnn.models.contact import Contact
-from deeprank_gnn.models.grid import MapMethod, Grid, GridSettings
-from deeprank_gnn.domain.storage import *
-
+from .structure import Atom, Residue
```

Comment:
You can't just remove the imports, they are used within the script.
Have you ran the tests/model/test_graph.py ?

Reply:
Indeed I removed only the unused imports. And instead of importing all functions/classes from modules (with `import *`), I copied explicitly functions/classes and then I removed only the ones not used (see the lines of code below)

Reply:
Sorry! I didn't seed the part below.